docs/conf.py

```diff
@@ -136,349 +106,163 @@
+# -- Options for LaTeX output --------------------------------------------------
 
 latex_elements = {
     # The paper size ('letterpaper' or 'a4paper').
-    #
     # 'papersize': 'letterpaper',
 
     # The font size ('10pt', '11pt' or '12pt').
-    #
     # 'pointsize': '10pt',
 
     # Additional stuff for the LaTeX preamble.
-    #
     # 'preamble': '',
-
-    # Latex figure (float) alignment
-    #
-    # 'figure_align': 'htbp',
 }
 
 # Grouping the document tree into LaTeX files. List of tuples
-# (source start file, target name, title,
-#  author, documentclass [howto, manual, or own class]).
+# (source start file, target name, title, author, documentclass [howto/manual]).
 latex_documents = [
-    (master_doc, 'edx-enterprise.tex', 'enterprise Documentation',
-     author, 'manual'),
+    ('index', 'enterprise.tex', u'Open edX Enterprise Service Documentation',
+     u'edX', 'manual'),
 ]
 
 # The name of an image file (relative to this directory) to place at the top of
 # the title page.
-#
 # latex_logo = None
 
 # For "manual" documents, if this is true, then toplevel headings are parts,
 # not chapters.
-#
 # latex_use_parts = False
 
 # If true, show page references after internal links.
-#
 # latex_show_pagerefs = False
 
 # If true, show URL addresses after external links.
-#
 # latex_show_urls = False
 
 # Documents to append as an appendix to all manuals.
-#
 # latex_appendices = []
 
-# It false, will not define \strong, \code, 	itleref, \crossref ... but only
-# \sphinxstrong, ..., \sphinxtitleref, ... To help avoid clash with user added
-# packages.
-#
-# latex_keep_old_macro_names = True
-
 # If false, no module index is generated.
-#
 # latex_domain_indices = True
 
 
-# -- Options for manual page output ---------------------------------------
+# -- Options for manual page output --------------------------------------------
 
 # One entry per manual page. List of tuples
 # (source start file, name, description, authors, manual section).
 man_pages = [
-    (master_doc, 'enterprise', 'enterprise Documentation',
-     [author], 1)
+    ('index', 'Open edX Enterprise Service', u'Open edX Enterprise Service Documentation',
+     [u'edX'], 1)
 ]
 
 # If true, show URL addresses after external links.
-#
 # man_show_urls = False
 
 
-# -- Options for Texinfo output -------------------------------------------
+# -- Options for Texinfo output ------------------------------------------------
 
 # Grouping the document tree into Texinfo files. List of tuples
 # (source start file, target name, title, author,
 #  dir menu entry, description, category)
 texinfo_documents = [
-    (master_doc, 'edx-enterprise', 'enterprise Documentation',
-     author, 'edx-enterprise', 'Your project description goes here',
-     'Miscellaneous'),
+    ('index', 'Open edX Enterprise Service', u'Open edX Enterprise Service Documentation',
+     u'edX', 'Open edX Enterprise Service', 'Open edX Enterprise Service',
+     'Miscellaneous'
```

Comment:
@mattdrayer Same idea - please use constants instead of plain strings where possible